docs/root/intro/version_history.rst

```diff
@@ -6,6 +6,9 @@ Version history
 * fault: removed integer percentage support.
 * stream: renamed the `RequestInfo` namespace to `StreamInfo` to better match
   its behaviour within TCP and HTTP implementations.
+* http: no longer adding whitespace when appending X-Forwarded-For headers. **Warning**: this is not
```

Comment:
Can you alpha order? I think you need to merge master anyway to pick up CI fixes.